src/config/configFile.ts

```diff
@@ -103,9 +110,16 @@ export class ConfigFile<T extends ConfigFile.Options> extends BaseConfigStore<T>
    * @param [throwOnNotFound = false] Optionally indicate if a throw should occur on file read.
    */
   public async read(throwOnNotFound = false): Promise<ConfigContents> {
```

Comment:
Can we add another param here to `updateCache` or something? There will be times we want to update from the filesystem I would imagine.

Reply:
I couldn't think of usecases where we'd want to.  We're always up to date while the process is running.